site/docs/guides/flowctl/edit-specification-locally.md

```diff
@@ -0,0 +1,122 @@
+
+Every *entity* (including active *tasks*, like captures and materializations, and static *collections*)
+has a globally unique name in the Flow catalog.
+
+For example, a given Data Flow may comprise:
+
+* A capture, `myOrg/marketing/leads`, which writes to...
+* Two collections, `myOrg/marketing/emailList` and `myOrg/marketing/socialMedia`, which are materialized as part of...
+* A materialization, `myOrg/marketing/contacts`.
+
+Using these names, you'll identify and pull the relevant specifications for editing.
+
+1. Authorize flowctl.
+
+   1. Go to the [CLI-API tab of the web app](https://dashboard.estuary.dev/admin/api) and copy your access token.
+
+   2. Run `flowctl auth token --token <paste-token-here>`
+
+2. Determine which entities you need to pull from the catalog. You can:
+
+   * Check the web app's **Captures**, **Collections**, and **Materializations** pages.
+  All published entities to which you have access are listed and may be searched.
+
+   * Run `flowctl catalog list`. This command returns a complete list of entities to which you have access.
+  You can refine by specifying a `--prefix` and filter by entity type:  `--captures`, `--collections`, `--materializations`, or `--tests`.
+
+    From the above example, `flowctl catalog list --prefix myOrg/marketing --captures --materializations` would return
+    `myOrg/marketing/leads` and `myOrg/marketing/contacts`.
+
+3. Pull the specifications you need by running `flowctl catalog pull-specs`:
+
+   * Pull one or more specifications by name, for example: `flowctl catalog pull-specs --name myOrg/marketing/emailList`
+
+   * Pull a group of specifications by prefix or type filter, for example: `flowctl catalog pull-specs --prefix myOrg/marketing --collections`
+
+  The source files are written to your current working directory.
+
+4. Browse the source files.
+
+  The directory structure will look slightly different depending on which specifications you pull.
+  Regardless, there will always be a top-level file called `flow.yaml` that *imports* all other nested YAML files.
+  These, in turn, contain the entity's specifications.
+
+  Specifications sharing a catalog prefix are written to the same YAML file for convenience.
+
+## Edit source files and re-publish specifications
+
+Next, you'll complete your edits, test that they were performed correctly, and re-publish everything.
+
+1. Open the YAML files that contain the specification you want to edit.
+
+2. Make changes. For guidance on how to construct Flow specifications, see the documentation for the task type:
+
+   * [Captures](../../concepts/captures.md#specification)
+   * [Collections](../../concepts/collections.md#specification)
+   * [Materializations](../../concepts/materialization.md#specification)
+   * [Derivations](../../concepts/derivations.md#specification)
+   * [Tests](../../concepts/tests.md)
+
+3. When you're done, run a generic test to make sure all the specifications are formatted correctly:
```

Comment:
nit: the `test` command is checking a lot more than just the formatting. And IDK if "generic" is pulling much weight:

```suggestion
3. As you work, you can run `flowctl catalog test --source flow.yaml` to test your changes.
```